README.md

```diff
@@ -90,21 +90,14 @@ mvn clean package -DskipTests -Dspark3
 mvn clean package -DskipTests -Dspark3.1.x
 ```
 
-### Build without spark-avro module
+### What about "spark-avro" module?
 
-The default hudi-jar bundles spark-avro module. To build without spark-avro module, build using `spark-shade-unbundle-avro` profile
+Previously, Hudi bundles were packaging (and shading) "spark-avro" module internally. However, due to multiple occasion 
+of it being broken b/w patch versions (most recent was, b/w 3.2.0 and 3.2.1) of Spark after substantial deliberation 
+we took a decision to let go such dependency and instead simply clone the structures we're relying on to better control 
```

Comment:
we can shorten this a bit and just make README have the actual steps to do here?

```
### What about "spark-avro" module?

Hudi versions 0.11 or later, no longer required `spark-avro` to be specified using `--packages`

```